config/application.rb

```diff
@@ -22,6 +22,10 @@ class Application < Rails::Application
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
 
+    config.eager_load_paths << "#{config.root}/app"
```

Comment:
don't think we need this - probably don't want to encourage classes in root of app.